user-guide/modules/ROOT/pages/getting-started.adoc

```diff
@@ -304,7 +304,7 @@ For instance:
 
 . After the download has completed in your *Downloads* folder, right-click on the compressed file and extract it to a folder.
 
-TIP: boost:build[] is the officially supported build system used by the Boost libraries.
+TIP: b2[] is the officially supported build system used by the Boost libraries.
```

Comment:
In contexts with a `[]`, `boost:build[]` is a macro that is replaced with links to the documentation of tools and libraries in boostorg. 

![image](https://github.com/cppalliance/site-docs/assets/5369819/059f288e-b831-4aa0-a7e5-c08f7ea14deb)

`b2[]` is just a literal `"b2[]"`

![image](https://github.com/cppalliance/site-docs/assets/5369819/32bb9d9e-9ca2-420b-b622-020e05ca0bcb)